example/marketing_program_member_add.json

```diff
@@ -0,0 +1,28 @@
+{
+	"meta": {
+		"topic": [
+			"marketing"
+		]
+	},
+	"recipient": [
+		"https://bhhs.hsfaffiliates.com/profile/card#me"
+	],
+	"payload": {
+		"type": "MarketingProgramMemberAdd",
+		"agent": {
```

Comment:
I noticed an Id field in the top level payload of `example/marketing_program_member_remove.json` but I don't see one here. How do we know which report this person is getting added to?

Reply:
added, thanks!